examples/gatsby/plugins/gatsby-plugin-mui-emotion/getEmotionCache.js

```diff
@@ -0,0 +1,5 @@
+import createCache from '@emotion/cache';
+
+export default function getEmotionCache() {
+  return createCache({ key: 'css', prepend: true });
```

Comment:
In the Next.js example, we don prepend. What should be our policy?

Reply:
Actually, you are right, we don't need it as in the examples we do not use JSS. Will update it.